sourcemash/frontend/static/logotext.svg

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Is there a reason we're using svg files? PNG is probably the way to go. (http://graphicdesign.stackexchange.com/questions/28778/what-file-type-should-i-use-for-a-logo-when-exporting-from-adobe-illustrator).

I can open the files in Pixelmator, crop the background and upload to your branch if you'd like.